sdk/appconfiguration/azure-appconfiguration-provider/CHANGELOG.md

```diff
@@ -4,10 +4,14 @@
 
 ### Features Added
 
+* Added support for `keyvault_credential`, `keyvault_client_configs`, and `secret_resolver` as `kwargs` instead of useing `AzureAppConfigurationKeyVaultOptions`.
```

Comment:
I see we use keyvault as single word in some places and key vault as two words in some other places.

Do we need to make them consistent?

Reply:
Typo "useing"

Reply:
Everything new should be keyvault. Did I miss any locations?

Reply:
Get it. So `key_vault_options` is legacy name. :)

Reply:
legacy name/object